content/_index.md

```diff
@@ -7,10 +7,26 @@ BookToC: false
 
 # The Linux Userspace API (UAPI) Group
 
-The userspace API ("uapi") group is a community for people with an interest in innovating how we build, deploy, and run modern Linux operating systems.
+The userspace API ("uapi") group is a cross-distro community for people with an interest in innovating how we build, deploy, run, and safely update modern Linux operating systems.
```

Comment:
I like plain language and "cross-distro" always seems off to me if you want to write a for a broader audience. I don't have any good alternatives that would get the point across though :)

Reply:
Let's go with "broad" then?

Reply:
I think we can use ''cross-distro'' if nobody shares my opinion on this. It still conveys the intentions of the group well for people in the distro community.

Reply:
I think cross-distro is wrong because we're not just cross-distro we're aiming to be cross-vendor. IOW, there might not be a distro associated with some interested projects in any meaningful sense. So I would just drop the qualifier completely. It doesn't bring new info imho.